### ISSUE TITLE
sdaps on Ubuntu 24.04, 23.10 seems broken

### ISSUE THREAD

**Issue by glor** [sdaps/sdaps#270]
There are no packages for anything newer than Jammy Jellyfish (22.04) and indeed I couldn't get the existing packages working.

I tried manually compiling sdaps but got into issues, because pkg-config could not find `py3cairo` (and after commenting that one out, `cairo` also could not be found) although both were installed.

In the end I installed `distrobox` and used an Ubuntu 22.04 container to get it working.

**Comment by benzea** [comment]
Yeah, I know … I have been very inactive lately and all the packages really need updating. The trouble is, I am somewhat out of touch with development as I am not using it anymore and development is basically driven by the fact that there are users out there relying on SDAPS.

Said differently, it really helps if people complain^Wsend me reminders that this project is useful to them!

**Comment by benzea** [comment]
That is also a call for help. Updating the packages shouldn't be hard, it just needs someone trying to do it.

After all, the fedora packages actually get build automatically for merge requests (against rawhide), and that is working fine!

**Comment by benzea** [comment]
OK, a Ubuntu 24.04 build is done, you can try it. Both the sdaps and sdaps-stable repository should work fine (the stable one might have somewhat broken escaping, but the python version in 24.04 still accepts it).

Fedora copr should also be good now.

**Comment by pythonmobile** [comment]
Thank you for your work, and it is useful!
